modules/agar/src/main/ts/ephox/agar/api/Mouse.ts

```diff
@@ -6,6 +6,15 @@ import { Chain } from './Chain';
 import { Step } from './Step';
 import * as UiFinder from './UiFinder';
 
+const click = Clicks.click({ });
```

Comment:
I feel like we should let people keep the opportunity to pass arguments in here themselves — can we either leave it as `click = Clicks.click` or `click = (args: ???, moreArgs: ???) => Clicks.click(args)(moreArgs)`

Reply:
As discussed on slack, I've made it so the settings can be passed as an optional second argument.